sdk/iot/Azure.Iot.Hub.Service/src/Generated/Models/ExportImportDevice.cs

```diff
@@ -18,19 +18,19 @@ public ExportImportDevice()
         }
```

Comment:
Should we make this class internal?

Reply:
Oh, this is for the bulk reg api, not for for the bulk reg job. Ignore this then.